pkgs/top-level/all-packages.nix

```diff
@@ -25993,6 +25993,8 @@ with pkgs;
 
   psqlodbc = callPackage ../development/libraries/psqlodbc { };
 
+  psqlodbc-unixodbc = callPackage ../development/libraries/psqlodbc { withUnixODBC = true; withLibiodbc = false; };
```

Comment:
Why add this extra name for it?

Reply:
Yeah, the more I think of it, the less it makes sense to have this + unixODBCDrivers.psql.